frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -83,7 +84,11 @@ export const ClusterServiceVersionRow = withFallback<ClusterServiceVersionRowPro
 });
 
 export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps> = (props) => {
-  const EmptyMsg = () => <MsgBox title="No Cluster Service Versions Found" detail="" />;
+  const emptyMsgDetail = <React.Fragment>
+    <div>No Operators are available for project <span className="co-clusterserviceversion-empty__state__namespace">{UIActions.getActiveNamespace()}</span></div>
```

Comment:
I'm trying to decide if this message is OK when you have items, but they're all hidden by a filter. How hard is it to show a different message in that case? We might need to wire through an additional prop from `MultiListPage` to know.

Reply:
I can leave the old message in this case?